int32 id 
int16 len
float64 lat
float64 lon
float32 yaw
float32 vel_n
float32 vel_e
